### PR TITLE
mirage 3.7.0 changes

### PR DESCRIPTION
**Pull request by hannesm**

### CODE REVIEW

**Comment by hannesm** [comment]
testruns of the unix modes are failing due to mirage-unix (since 4.0.0) no longer ignoring all the errors, but properly reporting them (and exiting with a non-zero code).

**Comment by hannesm** [comment]
(TODO maybe require `~min:"2.0.0" "mirage-time"` etc. in the unikernels, thus users with old mirage will get an opam conflict instead of a compilation error)

File: .travis.yml [hunk 1]
```diff
@@ -7,14 +7,14 @@ sudo: false
 env:
   global:
     - TESTS=false
-    - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
+    - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git#easy"
```
Comment (samoht) [inline .travis.yml]: do you still need this now that the changes have been released?

Comment (hannesm) [reply .travis.yml]: no, this is no longer needed

Comment (samoht) [reply .travis.yml]: can you rebase your change to master? It'd be nice to remove the deprecation warnings in that repo :-)

**Comment by hannesm** [comment]
rebased and raised lower mirage bound to 3.7.0

**Comment by hannesm** [comment]
CI failure due to the change of behaviour https://github.com/mirage/mirage-unix/commit/380a36daffdc7d9b5746db65a0d9f1f6c9063342 (I can't find any travis run where the testrun actually suceeded, there seem to be missing network devices)

**Comment by hannesm** [comment]
@samoht this is ready to be merged imho

**Comment by samoht** [comment]
Thanks!

**Pull request merged**